must-call-checker/src/main/java/org/checkerframework/checker/mustcall/SocketAccumulationFrames.astub

```diff
@@ -0,0 +1,41 @@
+// This version of the standard assumptions for sockets is intended for use without the -AnoAccumulationFrames
```

Comment:
Isn't this one for use _with_ accumulation frames?  I.e., in cases where accumulation frames are not disabled?